src/main/kotlin/translations/TranslationFiles.kt

```diff
@@ -110,12 +111,46 @@ object TranslationFiles {
         element.delete()
     }
 
+    fun findTranslationKeyForText(context: PsiElement, text: String): String? {
+        val module = context.findModule()
+            ?: throw IllegalArgumentException("Cannot add translation for element outside of module")
+        var jsonVersion = true
+        if (!TranslationSettings.getInstance(context.project).isForceJsonTranslationFile) {
+            val version =
+                context.mcVersion ?: throw IllegalArgumentException("Cannot determine MC version for element $context")
+            jsonVersion = version > MC_1_12_2
+        }
+
+        if (!jsonVersion) {
+            return null // todo: lang file support
```

Comment:
Honestly I think you can ignore lang files, the support for that format is basically a dangling remnant of now unsupported versions.

Reply:
[a6112d6](https://github.com/minecraft-dev/MinecraftDev/pull/2292/commits/a6112d6d6299f333762e64609930b7c71463b2c3)